nengo/cache.py

```diff
@@ -284,16 +285,21 @@ def _check_legacy_file(self):
         legacy_file = os.path.join(self.cache_dir, self._LEGACY)
         if os.path.exists(legacy_file):
             with open(legacy_file, 'r') as lf:
-                version = int(lf.read().strip())
+                text = lf.read()
+            try:
+                lv, pp = tuple(int(x.strip()) for x in text.split('.'))
```

Comment:
I assume the legacy version will always be an integer... I can't imagine we'll ever go so high that ints aren't sufficient. I just thought that `x.y` looked more like typical versions than having two separate things, or having a comma separating them (which is what I did initially).